### ISSUE TITLE
"Maximum call stack size exceeded" error with 2.0.0-alpha.14

### ISSUE THREAD

**Issue by jclark-dot-org** [cesarParra/apexdocs#37]
Decided to give the alpha I try (I could use the additional scope and annotation support), but I cannot get anything but an error:

```
> $ npm exec -- apexdocs-generate -s force-app/                                                            
node:internal/fs/utils:530
function getStatsFromBinding(stats, offset = 0) {
                            ^

RangeError: Maximum call stack size exceeded
    at getStatsFromBinding (node:internal/fs/utils:530:29)
    at Object.statSync (node:fs:1537:10)
    at DefaultFileSystem.isDirectory (/Users/jclark/Documents/dev/axia-core-utils/node_modules/@cparra/apexdocs/lib/service/file-system.js:7:19)
    at /Users/jclark/Documents/dev/axia-core-utils/node_modules/@cparra/apexdocs/lib/service/apex-file-reader.js:17:52
    at Array.forEach (<anonymous>)
    at Function.processFiles (/Users/jclark/Documents/dev/axia-core-utils/node_modules/@cparra/apexdocs/lib/service/apex-file-reader.js:15:27)
    at /Users/jclark/Documents/dev/axia-core-utils/node_modules/@cparra/apexdocs/lib/service/apex-file-reader.js:18:45
    at Array.forEach (<anonymous>)
    at Function.processFiles (/Users/jclark/Documents/dev/axia-core-utils/node_modules/@cparra/apexdocs/lib/service/apex-file-reader.js:15:27)
    at /Users/jclark/Documents/dev/axia-core-utils/node_modules/@cparra/apexdocs/lib/service/apex-file-reader.js:18:45

Node.js v17.2.0
```

I tried it in an empty project with a single apex class, with a single method (see below), and I get the same error.  Should I try with a different version of Node.js?
```
/**********************************************************
@group          core-utils
@description    Core apex utilities for use in all projects
@author         Jason Clark
@date           11/10/20
***********************************************************/
public class ApexUtil {

    /**
     * @Description Given a list of sObjects, return a set of IDs of the objects.
     * @param objs List of SObjects
     * @return Set of Ids for all sObjects
     * @example Set<Id> accountIds = ApexUtil.getIds(accounts);
     */
    public static Set<Id> getIds(List<SObject> objs) {
        Map<Id,SObject> tmpMap = new Map<Id,SObject>(objs);
        return tmpMap.keySet().clone();
    }
}
```

**Comment by cesarParra** [comment]
🤔 Interesting. That method is from node's fs library. Yeah it might be the node version, but I'll try to investigate

**Comment by cesarParra** [comment]
Sorry about the long wait on this, been unable to dedicate much time to this project lately, but now I should have some time for it going forwards.

The issue was around the way directories were being read and hopefully that is now addressed in the latest alpha version ([2.0.0-alpha.15](https://www.npmjs.com/package/@cparra/apexdocs/v/2.0.0-alpha.15))
